src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -52,7 +52,7 @@ public PersonCard(Person person, int displayedIndex) {
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
         phone.setText(person.getPhone().value);
-        address.setText(person.getAddress().value);
+        telegramHandle.setText(String.format("@%s", person.getTelegramHandle().value));
```

Comment:
Codecov angry

Reply:
Yes unfortunately we don't test ui here ;p